schemas/fluidity_options.rnc

```diff
@@ -3562,12 +3657,98 @@ prognostic_detector_options = detector_options_enabled_default
 diagnostic_detector_options = detector_options_enabled_default
 prescribed_detector_options = detector_options_disabled_default
 
+# Options for whether a field is to be included in particles.
+particle_options_enabled_default = 
+   (
+      ## Specify what is added to particles
+      element particles {
+         (
+            ## This field is included at each particle location.
+            element include_in_particles {
+	       element store_old_field {
+	       	       comment
```

Comment:
This needs a comment - "Store values of field at previous timestep/non-linear iteration, for use when updating particle attributes".